luigi2.h

```diff
@@ -2673,12 +2672,13 @@ int _UICodeMessage(UIElement *element, UIMessage message, int di, void *dp) {
 		if (code->hScroll) {
 			UIRectangle hScrollBarBounds = element->bounds;
 			if (~code->e.flags & UI_CODE_NO_MARGIN) hScrollBarBounds.l += UI_SIZE_CODE_MARGIN + UI_SIZE_CODE_MARGIN_GAP;
```

Comment:
Can i get rid of this, it's inconsistant with all gui frameworks, it looks weird. Why should the horizontal scrollbar start after the margin ? no editor/ide does this.